JiraPS/Public/Remove-JiraIssue.ps1

```diff
@@ -0,0 +1,108 @@
+function Remove-JiraIssue {
+    [CmdletBinding(
+        ConfirmImpact = 'High',
+        SupportsShouldProcess
+    )]
+    param (
+        [Parameter(
+            Mandatory,
+            ValueFromPipeline,
+            Position = 0
+        )]
+        [ValidateNotNullOrEmpty()]
+        [ValidateScript(
+            {
+                if (("JiraPS.Issue" -notin $_.PSObject.TypeNames) -and (($_ -isnot [String]))) {
+                    $errorItem = [System.Management.Automation.ErrorRecord]::new(
+                        ([System.ArgumentException]"Invalid Type for Parameter"),
+                        'ParameterType.NotJiraIssue',
+                        [System.Management.Automation.ErrorCategory]::InvalidArgument,
+                        $_
+                    )
+                    $errorItem.ErrorDetails = "Wrong object type provided for Issue. Expected [JiraPS.Issue] or [String], but was $($_.GetType().Name)"
+                    $PSCmdlet.ThrowTerminatingError($errorItem)
+                    <#
+                      #ToDo:CustomClass
+                      Once we have custom classes, this check can be done with Type declaration
+                    #>
+                }
+                else {
+                    return $true
+                }
+            }
+        )]
+        [Object[]]
+        $Issue,
+
+        [Switch]
+        [Alias("deleteSubtasks")]
+        $IncludeSubTasks,
+
+        [System.Management.Automation.CredentialAttribute()]
+        [System.Management.Automation.PSCredential]
+        $Credential,
```

Comment:
For compatibility with PSv3 and v4, this requires a default value (because of `[System.Management.Automation.CredentialAttribute()]`)

`$Credential = [System.Management.Automation.PSCredential]::Empty`

Reply:
This has been fixed in commit 639c6081df25c01715bc882440bf5d1b7ec453f8.